docs/data/getting-started/overview/overview.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Quick start
+description: Get started with Pigment CSS, a zero-runtime CSS-in-JS library.
+---
+
+# Quick start
+
+<Description />
+
+## Installation
+
+Pigment CSS has two category of packages. First one is to be imported and used in your source code. This includes the public API of the package. Second category is to be imported in your bundler config file to configure and actually be able to use Pigment CSS. We currently support [`Next.js`](https://nextjs.org/) (no Turbopack yet), [`Vite`](https://vite.dev/) and [`Webpack`](https://webpack.js.org/).
```

Comment:
```suggestion
Pigment CSS has two category of packages.
First one is to be imported and used in your source code.
This includes the public API of the package.
Second category is to be imported in your bundler config file to configure and actually be able to use Pigment CSS.
We currently support [`Next.js`](https://nextjs.org/) (no Turbopack yet), [`Vite`](https://vite.dev/) and [`Webpack`](https://webpack.js.org/).
```

Not important for this PR, but putting each sentence in a new line, makes the review so much easier. This is the convention we use in the Material UI's docs

Reply:
Will do in separate PRs. This wasn't even real content.